index.html

```diff
@@ -918,6 +930,47 @@ <h2>
         evolving privacy landscape and participate in the corresponding
         evolution of the API.
       </p>
+      <section>
+        <h3>
+          Design Considerations and Alternatives
+        </h3>
+        <p>
+          When reviewing the privacy characteristics of the Digital Credentials
+          API it is important to understand the goals of its design, which is
+          to offer a more secure and private user experience that matches the
+          flexibility of existing alternatives.
+        </p>
+        <p>
+          As new adopters are joining the rapidly emerging digital credentials
+          space, an important choice for implementation is the connection
+          between verifiers and wallets, i.e. the means by which a credential
+          exchange protocol is initiated and the user navigates from the
+          website they're browsing to their wallet application.
+        </p>
+        <p>
+          The most popular existing solutions for connecting verifiers and
+          wallets are QR codes and custom URL schemes, see
+          [[[presenting-credentials-on-the-web]]]. Both solutions have security
+          and privacy concerns (see the analysis in
+          [[[presenting-credentials-on-the-web]]] and [[[custom-schemes]]]) and
+          offer the user agent very little ability to intermediate on behalf of
+          the user, e.g. in the form of a permission prompt.
+        </p>
+        <p>
+          With adoption of digital credential technology being quickly driven
+          forward by consumer demand and regulatory mandates, we consider it
+          essential for user agents to offer an alternative to these harmful
+          technologies that is easy to use for web and wallet developers, is
+          compatible with existing credential exchange protocols and, most
+          importantly, significantly increases user privacy and security.
+        </p>
+        <p>
+          This API is not meant to inhibit the development of other Web APIs
+          that further push the boundaries on user privacy, for example an API
+          that more strictly enforces unlinkability for specific purposes such
+          as age verification.
```

Comment:
```suggestion
          This API does not inhibit the development of other APIs
          that further enhance user privacy. For example, an API
          that more strictly enforces unlinkability of age verification.
```

Reply:
I'm going to leave the short text we have here for now and we can follow up in more detail in #289